context/AuthContextProvider.tsx

```diff
@@ -60,7 +63,16 @@ export const AuthContextProvider = ({
       await account.createEmailPasswordSession(user.email, user.password);
       getUser();
       router.push('/weeklyPicks');
+      toast.custom(
```

Comment:
The unit test for logging in to shoe the toast notification should go in this unit test file. There's none right now, so we'll need to create one. Only add test case for what you are touching though.